microraiden/microraiden/client/channel.py

```diff
@@ -105,7 +105,7 @@ def topup(self, deposit):
         )
         self.client.web3.eth.sendRawTransaction(tx)
 
-        log.info('Waiting for topup confirmation event...')
+        log.debug('Waiting for topup confirmation event...')
```

Comment:
just a comment: it's a good practice to always include data you are trying to debug in the log message (here the message can be `Waiting for topup confirmation event... tx=0xdead...beef`)